ops/lib/__init__.py

```diff
@@ -123,46 +125,46 @@ def autoimport():
 
 def _find_all_specs(path: typing.Iterable[str]) -> typing.Iterator[ModuleSpec]:
     for sys_dir in path:
-        if sys_dir == "":
-            sys_dir = "."
+        if sys_dir == '':
+            sys_dir = '.'
         try:
             top_dirs = os.listdir(sys_dir)
         except (FileNotFoundError, NotADirectoryError):
             continue
         except OSError as e:
-            logger.debug("Tried to look for ops.lib packages under '%s': %s", sys_dir, e)
+            logger.debug(f"Tried to look for ops.lib packages under '{sys_dir}': {e}")
```

Comment:
All of these changes to the logging calls need to be reverted. f-strings should not be used in logging calls, because logging tools use the unformatted string to (e.g.) de-dupe messages.

Reply:
I saw in existing code base that there are many places that use f-strings for logging, so I did it to keep it consistent. I'll revert them.

Reply:
Indeed :disappointed: 

```
ops/pebble.py
1642:                logger.warning(f'Invalid I/O command {command!r}')
1705:                    logger.warning(f'Invalid I/O command {command!r}')

ops/storage.py
61:            logger.debug(f"Initializing SQLite local storage: {filename}.")

ops/model.py
988:        logger.debug(f"could not cast {raw} to IPv4/v6 address")
```

I'll open a different PR to fix those.